homeassistant/components/thermostat/homematic.py

```diff
@@ -108,21 +144,48 @@ def device_state_attributes(self):
                 "battery": self._battery,
                 "mode": self._mode}
 
+    @property
+    def min_temp(self):
```

Comment:
It means that this thermostat has an on and an off setting where it's not temperature controlled but hard open or closed.